lib/src/legacy/importer.ts

```diff
@@ -0,0 +1,274 @@
+// Copyright 2022 Google LLC. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+import {strict as assert} from 'assert';
+import {pathToFileURL, URL as NodeURL} from 'url';
+import * as fs from 'fs';
+import * as p from 'path';
+
+import {resolvePath} from './resolve-path';
+import {
+  fileUrlToPathCrossPlatform,
+  thenOr,
+  PromiseOr,
+  SyncBoolean,
+} from '../utils';
+import {
+  Importer,
+  ImporterResult,
+  LegacyAsyncImporter,
+  LegacyImporter,
+  LegacyImporterResult,
+  LegacyImporterThis,
+  LegacyPluginThis,
+  LegacySyncImporter,
+} from '../vendor/sass';
+
+/**
+
+ * Special URL protocols we use to for the legacy importer. These are generally
+ * not exposed to user code, but used internally to track
```

Comment:
Nit: incomplete?

Reply:
Yeah, this entire variable is left over from an abortive refactor.